app/controllers/opendata/agents/nodes/app/app_controller.rb

```diff
@@ -11,159 +11,159 @@ class Opendata::Agents::Nodes::App::AppController < ApplicationController
+  end
 
-    def set_file
-      set_app
-      filename = params[:filename]
-      filename.force_encoding("utf-8") if filename.present?
-      filename = "index.html" if filename.blank?
-      @appfile = @app.appfiles.find_by filename: filename
-    end
+  def set_file
+    set_app
+    filename = params[:filename]
+    filename.force_encoding("utf-8") if filename.present?
+    filename = "index.html" if filename.blank?
+    @appfile = @app.appfiles.find_by filename: filename
+  end
 
   public
-    def pages
-      Opendata::App.site(@cur_site).and_public
-    end
-
-    def index
-      @count          = pages.size
-      @node_url       = @cur_node.url
-      @search_path    = view_context.method(:search_apps_path)
-      @rss_path       = ->(options = {}) { view_context.build_path("#{view_context.search_apps_path}rss.xml", options) }
-      @tabs = []
-      Opendata::App.sort_options.each do |options|
-        if @cur_node.show_tab?(options[1])
-          @tabs << { name: @cur_node.tab_title(options[1]).presence || options[0],
-                     id: options[1],
-                     url: @search_path.call("sort" => options[1]),
-                     pages: pages.order_by(Opendata::App.sort_hash(options[1])).limit(@cur_node.limit || 10),
-                     rss: @rss_path.call("sort" => options[1])}
-        end
+  def pages
+    Opendata::App.site(@cur_site).and_public
+  end
+
+  def index
+    @count          = pages.size
+    @node_url       = @cur_node.url
+    @search_path    = view_context.method(:search_apps_path)
+    @rss_path       = ->(options = {}) { view_context.build_path("#{view_context.search_apps_path}rss.xml", options) }
+    @tabs = []
+    Opendata::App.sort_options.each do |options|
+      if @cur_node.show_tab?(options[1])
+        @tabs << { name: @cur_node.tab_title(options[1]).presence || options[0],
+                   id: options[1],
+                   url: @search_path.call("sort" => options[1]),
+                   pages: pages.order_by(Opendata::App.sort_hash(options[1])).limit(@cur_node.limit || 10),
+                   rss: @rss_path.call("sort" => options[1])}
       end
-
-      max = 50
-      @areas    = aggregate_areas(max)
-      @tags     = aggregate_tags(max)
-      @licenses = aggregate_licenses(max)
     end
 
-    def download
-      zipfilename = @app.create_zip
-      send_file zipfilename, type: "application/zip", filename: "#{@app.name}.zip",
-        disposition: :attachment, x_sendfile: true
-    end
+    max = 50
+    @areas    = aggregate_areas(max)
+    @tags     = aggregate_tags(max)
+    @licenses = aggregate_licenses(max)
+  end
 
-    def rss
-      @items = pages.order_by(released: -1).limit(100)
-      render_rss @cur_node, @items
-    end
-
-    def show_point
-      @cur_node.layout = nil
-      @mode = nil
+  def download
+    zipfilename = @app.create_zip
+    send_file zipfilename, type: "application/zip", filename: "#{@app.name}.zip",
```

Comment:
__[brakeman]__
* Model attribute used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

<!-- Comment by [SideCI](https://sideci.com) -->